src/Transition/InputRulesFiles/Rules9-5-0-to-9-6-0.md

```diff
@@ -4,4 +4,10 @@ Input Changes version 9.4.0 to 9.5.0
 This file documents the structural changes on the input of EnergyPlus that could affect interfaces, etc.
 This was previously an Excel workbook that made for very difficult version control, especially during busy times around code freezes.
 
-# Object Change: Dummy Placeholder
+# Object Change: Coil:Cooling:Water:DetailedGeometry
+
+Summary: Appendded new optional numeric input field *Design Water Inlet Temperature*.
+
+Field 1-24 remains the same.
+New Field F25 (N18): Design Water Inlet Temperature.  Leave blank or assign a typical chilled water supply temperature.
+
```

Comment:
Updated Input Rules File for adding new optional input field in Detailed Water Cooling Coil object.